infra/dlv/dlv-setup.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.!/bin/bash
+
+SSH="ssh -o StrictHostKeyChecking=no"
+SCP="scp -q -o StrictHostKeyChecking=no"
+
+function usage() {
+    echo "Usage: $0 -h vch-address" >&2
+    exit 1
+}
+
+while getopts "h" flag
+do
+    case $flag in
+
+        h)
+            # Optional
+            export DLV_TARGET_HOST="$OPTARG"
+            ;;
+
+        *)
+            usage
+            ;;
+    esac
+done
+
+shift $((OPTIND-1))
+
+if [ -z "${DLV_TARGET_HOST}" ]; then
+    usage
+fi
+
+DLV_BIN="$GOPATH/bin/dlv"
+
+if [ -z "${GOPATH}" -o -z "${GOROOT}" ]; then
```

Comment:
>  If the environment variable is unset, `GOPATH` defaults to a subdirectory named "`go`" in the user's home directory (`$HOME/go` on Unix, `%USERPROFILE%\go` on Windows), unless that directory holds a Go distribution. Run "`go env GOPATH`" to see the current `GOPATH`. 
― https://golang.org/cmd/go/#hdr-GOPATH_environment_variable

Instead of reading these variables directly, can we just call `go env GOPATH` so that we pick up the default value (if the user hasn't overriden it) and ensure we automatically pick up any changes go makes to the way this is resolved?